### ISSUE TITLE
Move notes from reproducibility to a new section

### ISSUE THREAD

**Issue by ItsPugle** [SamEm/Bug-Bot#18]
Whenever a BugHunter™️ `!addnote`-s, the note is added to the reproducibility section of the ticket. Would it be possible that when the first note is created for a ticket, a new section is formed named `**Notes**` and posted under that? 

For example, instead of this:
![no new section](https://d1ro8r1rbfn3jf.cloudfront.net/ms_100308/cjNMkoVNKJkAYdI0JXagmRsZa0hvRW/3cnbr2kgmt.png?Expires=1499068830&Signature=OgG9b~GD8WwlBYZA4NcxUz2uJhkxLAH~M8gSJCCsvymrEVFhq0wQnELQuVHk4PAIGtKcEkXL84Rme3ZNhbCUg0ucIydXcxKLR0-9aPK~N9ei1T3eCVwzgFfGoScgEHGRwFpjZZY-rB6zEKPAqUmU-e6N2WsfrF0Rz0ztPPlS4ZHGtHnfLFayU-aDY-yfTvYWPcP6K6PTcIQgITXN8OqRVzf7DGUI-oDF1u6SVCdrbgxOB~04FUHbhLfJV7bPL1izzjyyxm5bYqGADAVhZeQIec99Hf~8kviE8GkGWYOVv1mBH77lGtPsPi0c8SYKM2mrDXTDvn1sOxRbyDqS3Bo4Cg__&Key-Pair-Id=APKAJHEJJBIZWFB73RSA)
it would be this:
![new section](https://d1ro8r1rbfn3jf.cloudfront.net/ms_100308/LaPfBQubVY2jYmzYojay2FZLUvkCl2/2gmi8z1uk5.png?Expires=1499068842&Signature=c72NGp44Tioz2tQH7~YhtQiP5EFOmZ5dKN~QqHRnka~wuOH4CZHd5aO38TxpiTmApRBcDy8cwMrbeDz9PoQcgMJP93bZwZ07KiJXG08uSO--T2ARkPIuxoc3Bhwx5uy8MqGNmRLntEwID8zc9k8q0tobP82W6~q5hetudo0j2DkfddR4qH~-j37Mqxrxyw9fFPrMWx508OeAkf1U0oxDrRZ0KIZvbvX-Gjszr5rmC97jhRaIoEd0m7OilE0~aYe3A1rgBCVpC32vehKYJw2LeG~KuLvj9R80YQbin2bkKKgWhyJcAftF~y79X9ha~2FI9EMP0-tjWVwYqhifpGgZMg__&Key-Pair-Id=APKAJHEJJBIZWFB73RSA)

Ofcourse, with the actual green tick icons, not the default ✅  ones.

**Comment by Tarek1337** [comment]
Not needed since notes are getting removed from the queue, then it would be completely useless because the notes will appear only on the trello.

**Comment by ItsPugle** [comment]
@Tarek1337 > Notes aren't being removed from the BAQ? I saw that _someone_ suggested it, but I thought that the majority was against it and that Dabbit (or someone) was going to make it mroe strict. Unless I mistook something?

I saw this, but not a message from Dabbit or someone that is an official like 'yes, we're getting rid of notes on the BAQ.'
![image](https://user-images.githubusercontent.com/22669894/29023621-d8da9f82-7bb1-11e7-8fe7-294549540b11.png)
